### DISCUSSION TITLE
Multiple permalinks

### DISCUSSION THREAD

**Discussion by phber** [alshedivat/al-folio#2234] [Q&A] [answered]
Hi, I want '/about' to redirect to my home page: '/' . How would I go about adding this permalink, or in other words, how do I add multiple permalinks to the same page?

**Comment by george-gca** [comment, marked as answer]
We use `jekyll`, and `jekyll` [doesn't natively support this](https://github.com/jekyll/jekyll/issues/3368). Maybe you could do this with a [plugin](https://github.com/jekyll/jekyll-redirect-from).
